torch_xla/csrc/debug_util.cpp

```diff
@@ -120,6 +121,13 @@ void DebugUtil::SaveTensorsGraphInfo(const char* name,
       "XLA_SAVE_TENSORS_FILE", "", GetCurrentDevice().ordinal());
   if (!save_file.empty()) {
     static std::mutex lock;
+    if (format == DebugUtil::GraphFormat::kHlo && indices->size() > 0) {
+      // Dumping the HLO might access the placeholder data created during
+      // previous execution. We need to wait for last execution to finish before
+      // proceeding.
```

Comment:
I wonder what consequence we get if we don't wait, for example what error would you get when you run the python script